scripts/vim-patch.sh

```diff
@@ -399,7 +402,7 @@ show_vimpatches() {
   printf "\nVim patches missing from Neovim:\n"
 
   list_missing_vimpatches | while read vim_commit; do
-    if (cd "${VIM_SOURCE_DIR}" && git --no-pager  show --color=never --name-only "v${vim_commit}" 2>/dev/null) | grep -q ^runtime; then
+    if [[ "$(git -C "$VIM_SOURCE_DIR" diff-tree --no-commit-id --name-only  "${vim_commit}" -- runtime)" ]]; then
```

Comment:
why is this in `[[]]` ?

Reply:
Bash.  Not required though.

Reply:
Would prefer to avoid bashisms. If it's checking for non-empty, perhaps `test -n ...`

Reply:
`test -n` does not work, it also checks the exit code there.
I will look into improving this part, since it currently runs `git` for each patch again, which could also be done in one command maybe.
It is just used to eventually add the `(+runtime)` there.
But it does not appear to add much overhead after all (`20.71s user 35.62s system 97% cpu 57.733 total` without this still).